cmd/protoc-gen-gorums/gengorums/gorums.go

```diff
@@ -84,6 +95,26 @@ func genGorumsMethod(g *protogen.GeneratedFile, method *protogen.Method) string
 	panic(fmt.Sprintf("unknown method type %s\n", method.GoName))
 }
 
+func genStrictOrderingMethods(data servicesData, methodOptions ...*protoimpl.ExtensionInfo) {
+	g := data.GenFile
+	for _, service := range data.Services {
+		for _, method := range service.Methods {
+			if hasStrictOrderingOption(method, methodOptions...) {
+				fmt.Fprintf(os.Stderr, "processing %s\n", method.GoName)
+				g.P(genStrictOrderingMethod(g, method))
+			}
+		}
+	}
+}
+
+func genStrictOrderingMethod(g *protogen.GeneratedFile, method *protogen.Method) string {
```

Comment:
This func is almost identical to `genGorumsMethod`. Please use this func instead. The method name in the error message should uniquely identify where the problem is. Reaching the panic can only happen if someone is playing around with adding more options and forgot to update some of the structs.